tests/utils/command_test_utils.go

```diff
@@ -0,0 +1,253 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/google/go-cmp/cmp"
+	"sigs.k8s.io/yaml"
+
+	schema "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+)
+
+// AddEnv creates and returns a specifed number of env attributes in a schema structure
+func AddEnv(numEnv int) []schema.EnvVar {
```

Comment:
func can be unexported, used only within the pkg

pls check for all funcs in this file.